evm/src/cpu/modfp254.rs

```diff
@@ -0,0 +1,53 @@
+use itertools::izip;
+use plonky2::field::extension::Extendable;
+use plonky2::field::packed::PackedField;
+use plonky2::field::types::Field;
+use plonky2::hash::hash_types::RichField;
+use plonky2::iop::ext_target::ExtensionTarget;
+
+use crate::constraint_consumer::{ConstraintConsumer, RecursiveConstraintConsumer};
+use crate::cpu::columns::CpuColumnsView;
+
+// Python:
+// >>> P = 21888242871839275222246405745257275088696311157297823662689037894645226208583
```

Comment:
hmm according to https://neuromancer.sk/std/bn/bn254, the prime is 
```py
In [1]: 0x2523648240000001BA344D80000000086121000000000013A700000000000013
Out[1]: 16798108731015832284940804142231733909889187121439069848933715426072753864723
```

Reply:
Jacqui's prime is the correct one.
(There are two BN curves that go by the name `bn254`, the one used by Ethereum is [this one](https://pkg.go.dev/github.com/consensys/gnark-crypto/ecc/bn254), which is different from the curve referenced in neuromancer.sk)

Reply:
oh wow! sorry for the remark then :)